addon/components/project-selector/template.hbs

```diff
@@ -85,13 +85,21 @@
         {{/if}}
         {{#if (or willCreateComponent willMoveToNode)}}
             <p class="m-t-lg">
-                {{t "eosf.components.moveToProject.convertOrCopyMessage"}}
-            </p>
-            <p class="text-danger m-t-sm">
-                {{#unless isProjectPublic}}
-                    {{t "eosf.components.moveToProject.noLongerPublicWarning"}}
-                {{/unless}}
+                {{#if isChildNode}}
+                    {{t "eosf.components.moveToProject.convertOrCopyMessage.component"}}
+                {{else}}
+                    {{t "eosf.components.moveToProject.convertOrCopyMessage.project"}}
+                {{/if}}
```

Comment:
One of these will always display once the user has selected where to move the file, either to a new component under an existing node (`willCreateComponent`) or to an existing node (`willMoveToNode`)